effects.cc

```diff
@@ -91,14 +91,17 @@ void Effects_manager::add_text(
  *  @param y        y coord. on screen.
  */
 
-void Effects_manager::add_text(
+inline void Effects_manager::add_text(
```

Comment:
It looks like `add_text` is referred to in other places, and making the function `inline` breaks the build. From the [failed build log](https://travis-ci.org/github/exult/exult/jobs/740989003#L2966):

```
/home/travis/build/exult/exult/gamewin.cc:2122: undefined reference to `Effects_manager::add_text(char const*, int, int)'
```

I was able to reproduce this locally, and undoing this change fixed the problem.

```suggestion
void Effects_manager::add_text(
```

Reply:
> It looks like `add_text` is referred to in other places, and making the function `inline` breaks the build. From the [failed build log](https://travis-ci.org/github/exult/exult/jobs/740989003#L2966):
> 
> ```
> /home/travis/build/exult/exult/gamewin.cc:2122: undefined reference to `Effects_manager::add_text(char const*, int, int)'
> ```
> 
> I was able to reproduce this locally, and undoing this change fixed the problem.

True, although in such case I wonder if moving the one-liner to header file to keep it inline wouldn't be a better option.
I will check it later today.

Reply:
Moving itt to the header would require including a bunch of headers. Better to just remove the inline.